google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableTracer.java

```diff
@@ -93,4 +93,12 @@ public void grpcChannelQueuedLatencies(long queuedTimeMs) {
   public void grpcMessageSent() {
     // noop
   }
+
+  /**
+   * Set the customer specified deadline so that it can be exported in a metric. This will be
```

Comment:
Let's make the description more specific, something like "record the deadline when the request is sent to Bigtable server ..."